src/config/gremlins/gremlins.tsx

```diff
@@ -154,21 +154,21 @@ export const gremlins: GremlinList<GremlinId> = {
         ) {
           return {
             capacityChange: -1,
-            userStoryChance: -10,
+            userStoryChange: -10,
             title:
               'Emergency Story Mid-Sprint effect reduce by either the effect of Ongoing Backlog Refinement or good Strategic work with the Product Owner',
           };
         }
 
         return {
           capacityChange: -2,
-          userStoryChance: -10,
+          userStoryChange: -10,
```

Comment:
Here you made a typo. Effects do not know a `userStoryChance` property. 

This is fine for typescript since:

1. An object is allowed to have additional fields not known to it's type
   ```ts
   type Test = { one: number };
   const t: Test = { one: 1, two: 2 };
   ```
2. Since we [moved to multi-purpose effects](https://github.com/mlevison/high-performance-teams-game/issues/85) it's no error to have an effect with just a `title` (and typos) since all properties are optional

Reply:
Good catch. This really does hint why I find myself more comfortable in languages with a stronger typing model :-)